### DISCUSSION TITLE
Invalid Syntax in Telebot Ddata Visualisation

### DISCUSSION THREAD

**Discussion by vincentvaldinata** [teamalgoritma/community#374] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/76637079/193399875-db2f33d2-fd65-4860-9139-d71dcb5dea6c.png)

invalid syntax, i dont find the eror

**Comment by kevbow** [comment, marked as answer]
Dear @vincentvaldinata,

Silakan berikan tanda kutip string pada hex code yang Anda gunakan untuk menentukan warna. Jika Anda menggunakan hex code tanpa kutip maka tanda pagar akan membuat apapun yang dimasukkan setelahnya menjadi comment, hal ini menyebabkan fungsi Anda kekurangan tanda kurung tutup sehingga muncul error invalid syntax.

Semoga membantu.
